js/background.js

```diff
@@ -230,6 +231,16 @@
         var details = ev.contactDetails;
 
         var id = details.number;
+
+        if (id === textsecure.storage.user.getNumber()) {
+          // special case for syncing details about ourselves
+          if (details.profileKey) {
+            console.log('Got sync message with our own profile key');
+            storage.put('profileKey', profileKey);
+          }
+          return ev.confirm();
```

Comment:
I think we should eliminate the `return` here, since it will prevent the creation of a local `Conversation` for yourself. And I'm pretty sure we rely on it for conversations with yourself, and display of the members in a group.

Reply:
done